npm/ng-packs/packages/theme-shared/src/lib/components/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,32 @@
+import { AbstractNgModelComponent } from '@abp/ng.core';
+import { Component, EventEmitter, forwardRef, Injector, Input, Output } from '@angular/core';
+import { CheckboxControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+@Component({
+  selector: 'abp-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => CheckboxComponent),
+      multi: true,
+    },
+  ]
+})
+export class CheckboxComponent extends AbstractNgModelComponent {
```

Comment:
component class names do not have a naming convention between FormInputComponent and CheckboxComponent. Please make FormCheckboxComponent or please make FormInputComponent to InputComponent